packages/ui/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactText } from 'react';
+import { Badge as ChakraBadge, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+type BadgeSize = 'small' | 'large';
+
+type BadgeColor = 'default' | 'brand' | 'blue' | 'green' | 'orange' | 'red';
+
+export interface BadgeProps extends HTMLChakraProps<'div'> {
+  children: ReactText | ReactText[];
+  /**
+   * Color variant of the badge
+   */
+  color?: BadgeColor;
+  /**
+   * Determines badge's paddings, font size, line height and border radius.
+   */
+  size?: BadgeSize;
+}
+
+export const smallStyleProps = {
+  padding: '2px 10px',
+  borderRadius: '10px',
+  lineHeight: '16px',
+  fontSize: '12px',
+};
+
+export const largeStyleProps = {
+  padding: '2px 12px',
+  borderRadius: '12px',
+  lineHeight: '20px',
+  fontSize: '14px',
+};
+
+export const Badge = forwardRef<BadgeProps, 'div'>(({ size = 'small', color = 'default', children, ...props }, ref) => {
+  const sizeProps = size === 'small' ? { ...smallStyleProps } : { ...largeStyleProps };
+
+  return (
+    <ChakraBadge
+      textTransform="none"
+      size={size}
+      colorScheme={color}
```

Comment:
When `color` prop is equal to `'default'` you need to pass `gray` as the `colorScheme`. Currently, for the default badge, you have this CSS
![CleanShot 2021-06-28 at 23 19 55@2x](https://user-images.githubusercontent.com/22938071/123705550-699c1580-d867-11eb-95de-a3b689fc22a1.png)